node/network/bridge/src/lib.rs

```diff
@@ -295,7 +302,7 @@ impl<N, AD> NetworkBridge<N, AD> {
 impl<Net, AD, Context> Subsystem<Context> for NetworkBridge<Net, AD>
 	where
 		Net: Network + Sync,
-		AD: validator_discovery::AuthorityDiscovery,
+		AD: validator_discovery::AuthorityDiscovery + Clone,
```

Comment:
Can't we use the first one and then use it with an `Arc`?

Reply:
I guess we could, but the Arc is not necessary as the authority discovery service is an mpsc Sender which is perfectly clonable. The problem is, I need it as a trait object and that cannot be `Clone`.

Reply:
In that case, ignore :) there is no point in doing what I suggested.